docfx.json

```diff
@@ -105,6 +118,18 @@
       },
       "ROBOTS": {
         "api/**.yml": "NOINDEX, NOFOLLOW"
+      },
+      "ms.author": {
+        "_csharplang/spec/*.md": "wiwagn"
+      },
+      "topic": {
+        "_csharplang/spec/*.md": "reference"
+      },
+      "ms.date": {
+        "_csharplang/spec/*.md": "07/01/2017"
+      },
+      "open_to_public_contributors": {
```

Comment:
the edit button is still showing up. not sure if we need a full rebuild to make it work.